recipe/patches/0001-fix-binom_test-to-binomtest.patch

```diff
@@ -0,0 +1,1040 @@
+From 71d06202daea1ef9805ac8309dd6a0b2a3a4fd84 Mon Sep 17 00:00:00 2001
+From: Lorenzo Pirritano <lpirritano@anaconda.com>
+Date: Tue, 25 Jun 2024 14:08:19 +0200
+Subject: [PATCH 1/2] fix binom_test to binomtest
+
+The function `scipy.stats.binom_test()` was removed from SciPy 1.12.0
+
+See https://stackoverflow.com/questions/78090426/cannot-import-name-binom-test-from-scipy-stats-error-when-using-borutashap
+
+---
+ alibi_detect/cd/base.py | 590 +++++++++++++++++++++++++---------------
+ 1 file changed, 371 insertions(+), 219 deletions(-)
+
+diff --git a/alibi_detect/cd/base.py b/alibi_detect/cd/base.py
+index a3987e3..149a0ed 100644
+--- a/alibi_detect/cd/base.py
++++ b/alibi_detect/cd/base.py
+@@ -7,7 +7,7 @@ from alibi_detect.base import BaseDetector, concept_drift_dict, DriftConfigMixin
+ from alibi_detect.cd.utils import get_input_shape, update_reference
+ from alibi_detect.utils.frameworks import has_pytorch, has_tensorflow
+ from alibi_detect.utils.statstest import fdr
+-from scipy.stats import binom_test, ks_2samp
++from scipy.stats import binomtest, ks_2samp
+ from sklearn.model_selection import StratifiedKFold
+ 
+ if has_pytorch:
+@@ -20,24 +20,24 @@ logger = logging.getLogger(__name__)
+ 
+ 
+ class BaseClassifierDrift(BaseDetector):
+-    model: Union['tf.keras.Model', 'torch.nn.Module']
++    model: Union["tf.keras.Model", "torch.nn.Module"]
```

Comment:
A lot of these changes are just formatting changes. Can you undo these and only change what's needed please?

Reply:
Ah sorry, I forgot to disable the formatting. Done, thanks for the comment!